x-pack/plugins/task_manager/server/queries/mark_available_tasks_as_claimed.ts

```diff
@@ -15,23 +15,6 @@ import {
   MustNotCondition,
 } from './query_clauses';
 
-export function taskWithLessThanMaxAttempts(type: string, maxAttempts: number): MustCondition {
```

Comment:
I noticed a few lingering references to search-related things regarding tasks running too many attempts.  I believe this got resolved in https://github.com/elastic/kibana/pull/152841; though not sure if that applies to recurring tasks.  @mikecote @ymao1 ???  In any case, this function was no longer being used, so figured I might as well delete it.

Reply:
Yea I don't think we enforced anything with max attempts for recurring task types.

Reply:
+1 shouldn't be used for recurring tasks, only ad-hoc (one time) tasks